template/README.md

```diff
@@ -21,6 +21,35 @@ You may change the hashistack configuration or add aditional pre and post steps
 You may edit the `99-override.hcl` or add your own.
 Any valid configuration added to these directories will be added to their respective services' configuration, in lexical order.
 
+#### Config variations
+We provide some config variations as additional functionality, which you may want to use.
+Current supported variations:
+```text
+* ACL default policy for consul (deny or allow)
+```
+To choose other config variation you need to override ansible variables. There are two options how to override ansible variables:
+* First option via `ANSIBLE_AGRS`
+```bash
+ANSIBLE_ARGS='--extra-vars "consul_agent_acl_default_policy=deny"' vagrant up --provision
+```
+* Second option via prestart, adding fact in `vagrant/conf/ansible/playbooks/prestart/0-example.yml`
```

Comment:
```suggestion
* Second option via prestart, adding fact in `vagrant/conf/ansible/playbooks/prestart/0-acl.yml`
```

Reply:
What do you think?

Reply:
I think we will have several extra variables that will be NOT necessary associated with ACL functionality. So, it is the user's choice of how to name prestart `yml` script file. Considering this concrete example, I would suggest having one prestart script file `0-example.yml` which will contain an example of how to override extra vars (facts)

Reply:
I disagree. Yes, we will have many features. 0-example.yml gives no visual indication on what it is. However 0-acl.yml and 0-enterprise.yml etc wil definitly have that impact. The files can be commented by default. All you need to do is uncomment them? or they may reside in a sub-folder ( examples ?) ready to go?

Reply:
Would you like that I create `vagrant/conf/ansible/playbooks/prestart/0-acl.yml` with commented example? I would also bring the user's attention on the prestart script sequence and maybe name it `1-acl.yml`, WDYT?

Reply:
Yes.

Reply:
done